lib/node_modules/@stdlib/math/base/special/nonfibonacci/src/main.c

```diff
@@ -0,0 +1,82 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <stdint.h>
+ 
+#include "stdlib/constants/float64/nan.h"
+#include "stdlib/constants/float64/phi.h"
+#include "stdlib/constants/float64/pinf.h"
+#include "stdlib/math/base/assert/is_integer.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/floor.h"
+#include "stdlib/math/base/special/ln.h"
+
+#define SQRT5 2.23606797749979
+#define LN_PHI 0.48121182506
+
+/**z
+ * Computes the nth non-Fibonacci number.
+ *
+ * @param x    input value
+ * @return	    output value
+ *
+ * @example
+ * double y = stdlib_base_nonfibonacci( 2 );
+ * // returns 6.0
+ *
+ * @example
+ * double y = stdlib_base_nonfibonacci( 1 );
+ * // returns 4.0
+ *
+ * @example
+ * double y = stdlib_base_nonfibonacci( 3 );
+ * // returns 7.0
+ * 
+ * @example
+ * double y = stdlib_base_nonfibonacci( NaN );
+ * // returns NaN
+ * 
+ * @example
+ * double y = stdlib_base_nonfibonacci( 3.14 );
+ * // returns NaN
+ * 
+ * @example
+ * double y = stdlib_base_nonfibonacci( -1 );
+ * // returns NaN
+ */
+double stdlib_base_nonfibonacci(const int32_t n)
```

Comment:
Please add spaces around parentheses here and below to match our code conventions.